core/controllers/skill_editor.py

```diff
@@ -253,14 +253,24 @@ def delete(self, skill_id):
 class SkillDataHandler(base.BaseHandler):
     """A handler for accessing skills data."""
 
+    URL_PATH_ARGS_SCHEMAS = {
+        'skill_ids': {
+            'schema': {
+                'type': 'custom',
+                'obj_type': 'JsonEncodedInString'
+            }
+        }
```

Comment:
Can we validate if the skill IDs are in correct format?

Reply:
But the string which is getting passed can be a combination of more than one skill ID, so I couldn't manage to validate the format for each skill ID.
So I took reference from [#13961](https://github.com/oppia/oppia/pull/13961#pullrequestreview-772370641) where a similar kind of issue was discussed before.

Reply:
Ok, right.